bin/concert

```diff
@@ -27,37 +28,97 @@ LOG = logging.getLogger(__name__)
 # Start of enabling importing python files with "await" outside of "async def" functions
 
 
-class AsyncLoader(importlib.machinery.SourceFileLoader):
-    """
-    Normal SourceFileLoader with the capability of dealing with `await' outside of `async def'
-    functions. The trick is the flag ast.PyCF_ALLOW_TOP_LEVEL_AWAIT and eval() instead of exec()
-    used by Python's :class:`_LoaderBasics` class.
+def is_import(node):
```

Comment:
move this into _eval_nodes, nothing here is supposed to be accessed from outside.